test/inductor/test_flex_attention.py

```diff
@@ -3994,7 +4093,7 @@ def generate_test_inputs(self, full_seq_len: bool, device):
     @supported_platform
     @common_utils.parametrize("full_indices", [False, True])
     def test_from_kv_blocks(self, full_indices: bool):
-        device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+        device = torch.device(self.device if HAS_GPU else "cpu")
```

Comment:
Could this be simplified to `device = torch.device(self.device)`?
`test_device` already passed a `cpu` check at the beginning.